test/BMTFile.ts

```diff
@@ -0,0 +1,181 @@
+import {
+  Chunk,
+  fileInclusionProofBottomUp,
+  getSpanValue,
+  makeChunkedFile,
+  Utils,
+} from '@fairdatasociety/bmt-js'
+import { expect } from 'chai'
+import { ethers } from 'hardhat'
+import { BMTFile } from '../typechain'
```

Comment:
I have problem again with the typechain import. have you rebased this branch to the master well?

Reply:
ah, now I needed `npx hardhat compile` and it is fine